test/e2e/storage/utils/utils.go

```diff
@@ -818,3 +819,17 @@ func GetSizeRangesIntersection(first e2evolume.SizeRange, second e2evolume.SizeR
 	}
 	return "", fmt.Errorf("intersection of size ranges %+v, %+v is null", first, second)
 }
+
+// GetVolumeAttachment attempts to get the associated VolumeAttachment for the passed in PV.
+// It returns the VolumeAttachment, assuming one is found, and any generated errors
+func GetVolumeAttachment(cs clientset.Interface, pv *v1.PersistentVolume, provisioner, nodeName string) (*storagev1.VolumeAttachment, error) {
```

Comment:
Is `GetVolumeAttachment` needed for this PR? I don't see it used in snapshottable.go

Reply:
It's no longer specifically used. It'd still be nice to have, but I can simplify this PR, and then submit a cleanup one to include GetVolumeAttachment.

Reply:
I went ahead and broke these commits off to separate PRs. The one to use generic VolumeAttachments is at https://github.com/kubernetes/kubernetes/pull/102262 .